Cargo.toml

```
@@ -2,7 +2,7 @@
 authors = ["Jorge Aparicio <japaricious@gmail.com>"]
 name = "compiler_builtins"
 version = "0.1.135"
-license = "MIT/Apache-2.0"
+license = "Apache-2.0 WITH LLVM-exception AND MIT AND (MIT OR Apache-2.0)"
```

```suggestion
license = "MIT AND Apache-2.0 WITH LLVM-exception AND (MIT OR Apache-2.0)"
```

(Minor consistency/readability nit, not a blocker.)

---

Agreed, made this change